synchronizer/templates/validation.html

```diff
@@ -25,7 +25,13 @@
                     {% for worklog in worklogs %}
                     <tr class="{% if worklog.is_valid %} table-success {% else %} table-danger {% endif %}">
                         <th scope="row">{{loop.index}}</th>
-                        <td>{{worklog.comment}}</td>
+                        <td>{{worklog.comment}}
+                            {% if not worklog.is_valid %}
+                                <div>
+                                    <span class="badge badge-danger">Error: Issue id is not exists in {{ connector_name }}.</span>

```

Comment:
This is not an only case when issue will be invalid. At least we have second case: when there is no issue ID

Reply:
Also ID is abbreviation, should be UPPERCASE